docs/reference/api.md

```diff
@@ -62,6 +62,13 @@ Whether to build in production mode. -->
 
 Log level.
 
+#### `options.strict`
```

Comment:
only when i was writing the documentation I saw the code above:

```
<!-- #### `options.production`
- Type: `boolean`
- Default: `false`
Whether to build in production mode. -->
```

so maybe this flag should be named `--production` instead? 🤷

Reply:
No, you are right. The original purpose of production is to distinguish the environment for the build process